src/datasets/config.py

```diff
@@ -155,3 +150,9 @@
 DATASET_ARROW_FILENAME = "dataset.arrow"
 DATASET_INDICES_FILENAME = "indices.arrow"
 DATASET_STATE_JSON_FILENAME = "state.json"
+DATASET_INFO_FILENAME = "dataset_info.json"
+DATASETDICT_INFOS_FILENAME = "dataset_infos.json"
+LICENSE_FILENAME = "LICENSE"
+METRIC_INFO_FILENAME = "metric_info.json"
+
+MAX_DIRECTORY_NAME_LENGTH = 255
```

Comment:
```suggestion
MAX_DIRECTORY_NAME_LENGTH = 255  # to keep some paths readable (this is unrelated to the windows path length limit of 255, which you should disable to use this library)
```
I'd add this minor comment now that this config field is exposed in config.py

Reply:
I have renamed the variable to avoid confusion. The comment would not be visible to the end user by importing `datasets.config` and using tab completion.